### PR TITLE
[UPDATES] 

### PR DESCRIPTION
**Pull request by Mycroft-Studios**
updates Identity, Vehicleshop and Jobs. Adds esx_property

### CODE REVIEW

File: recipe.yaml [hunk 1]
```diff
@@ -84,7 +84,7 @@ tasks:
 
   # Download ESX resources
   - action: download_github
-    src: https://github.com/esx-framework/es_extended
```
Comment (tabarra) [inline recipe.yaml:87]: Nope, need to stick to the main esx, what changed compared to yours?

Comment (Mycroft-Studios) [reply recipe.yaml:87]: Ah, that was for debugging and forgot to change it back

Comment (Mycroft-Studios) [reply recipe.yaml]: Fixed

File: recipe.yaml [hunk 2]
```diff
@@ -238,6 +238,11 @@ tasks:
     ref: ee9c916ebf955abefa3bfdbd20a773122b865a22
     dest: ./resources/[esx]/esx_holdup
 
+  - action: download_github
+    src: https://github.com/esx-framework/esx_property
+    ref: 1475b201c01ff0cf9c4433e84da9444a10249312
+    dest: ./resources/esx_property
```
Comment (RockySouthpaw) [inline recipe.yaml:244]: Shouldnt this be 
`./resources/[esx]/esx_property`

**Review by tabarra — CHANGES_REQUESTED** [review]
Absolutely no for the ref changes to master... this 100% defeats the prupose of a ref, and breaks things all the time (for instance the crap you did to the legacy branch)

**Comment by tabarra** [comment]
This PR kinda fell out of control and got abandoned.

**Pull request closed without merging**